test_unstructured/partition/test_auto.py

```diff
@@ -432,15 +429,13 @@ def test_auto_with_page_breaks():
     assert PageBreak() in elements
 
 
-@pytest.mark.skipif(is_in_docker, reason="Skipping this test in Docker container")
 def test_auto_partition_epub_from_filename():
```

Comment:
@awalker4 I don't think my tests in instruction would make any difference, here is an unittest with `partition`